Model/Export/ProductExportManager.php

```diff
@@ -668,4 +675,19 @@ private function validateBarcode(string $attributeValue): ?string
 
         return $attributeValue;
     }
+
+    /**
+     * @param string $description
+     * @return string
+     */
+    private function formatProductDescription(string $description): string
```

Comment:
```suggestion
    static public function formatProductDescription(string $description): string
```

can you add unit test for this function only with dataProvider? We will probably need it in future